### DISCUSSION TITLE
Move Crackme to a main part of the website

### DISCUSSION THREAD

**Discussion by p2635** [OWASP/mastg#2261] [Ideas]
I literally just found out about Crackme, but only after browsing the repo. It's a wonderful resource but it's hidden within the docs! https://mas.owasp.org/MASTG/Tools/0x08b-Reference-Apps/?h=crackme. What a shame! For someone like me, it's an exciting initiative, so I'm opening up this discussion because maybe we can move this somewhere more prominent. Where? I honestly don't know yet, the main nav bar or something? Let's chat about it.

This is a great starting point for me to start learning about appsec and get more hands on. Crackme sounds like a project on its own!

**Comment by angietechcafe** [comment]
Hi there, I also believe this is a great idea. Maybe Crackme could be featured as a tool of the month? Or something that will attract people to learn more about Crackme. I would love to contribute to this idea as well.

**Reply by cpholguera** [reply]
Hi @angieintech, thanks for your feedback! We've just released a new tab for the crackmes today:

https://mas.owasp.org/crackmes

I really like your idea of some "crackme/tool of the month". That reminded me of something we were discussing recently: maybe implementing a top 3(?) ranking of tools. As you might know we have a MAS Testing Tools chapter:

https://mas.owasp.org/MASTG/Tools/0x08a-Testing-Tools/

These tools are used across the MASTG tests and using some Automation we could retrieve the most used ones for example and build a "top ranking" which can be published regularly.

Please feel free to open new "Ideas" discussions for any other ideas you might have. Thank you 😊

**Comment by cpholguera** [comment]
See https://mas.owasp.org/crackmes/
